Simplified/Accounts/AgeCheck/NYPLAgeCheckViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  NYPLAgeCheckViewController.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2021-02-22.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import UIKit
+
+class NYPLAgeCheckViewController: UIViewController {
+
+  // Constants
+  let textFieldHeight: CGFloat = 40.0
+  
+  fileprivate var birthYearSelected = 0
+  
+  weak var ageCheckDelegate: NYPLAgeCheckValidationDelegate?
+  
+  init(ageCheckDelegate: NYPLAgeCheckValidationDelegate) {
+    self.ageCheckDelegate = ageCheckDelegate
+    
+    super.init(nibName: nil, bundle: nil)
+  }
+  
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    self.setupView()
+  }
+  
+  // We need to fail the age check because user can swipe down to dismiss the view controller in iOS 13+
+  deinit {
+    if !(ageCheckDelegate?.ageCheckCompleted ?? false) {
+      ageCheckDelegate?.ageCheckFailed()
+    }
+  }
+  
+  @objc func completeAgeCheck() {
+    guard ageCheckDelegate?.isValid(birthYear: birthYearSelected) ?? false else {
+      return
+    }
+    
+    ageCheckDelegate?.ageCheckCompleted(birthYearSelected)
+    ageCheckDelegate?.ageCheckCompleted = true
+    dismiss(animated: true, completion: nil)
+  }
+  
+  // MARK: - UI
+  
+  func updateBarButton() {
+    rightBarButtonItem.isEnabled = ageCheckDelegate?.isValid(birthYear: birthYearSelected) ?? false
+  }
+  
+  @objc func hidePickerView() {
+    self.view.endEditing(true)
+  }
+  
+  func setupView() {
+    self.title = NSLocalizedString("Age Verification", comment: "Title for Age Verification")
+    
+    if #available(iOS 13.0, *) {
+      view.backgroundColor = UIColor.systemGray6
+    } else {
+      view.backgroundColor = .white
+    }
+    
+    navigationItem.setRightBarButton(rightBarButtonItem, animated: true)
+    
+    inputTextField.translatesAutoresizingMaskIntoConstraints = false
+    titleLabel.translatesAutoresizingMaskIntoConstraints = false
+    
+    view.addSubview(inputTextField)
+    view.addSubview(titleLabel)
+    
+    inputTextField.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
```

Comment:
we're already setting leading and trailing constraints at line 82 so we don't need a center constraint